$OpenBSD: patch-src_app_app_cc,v 1.3 2019/01/06 12:15:50 jasper Exp $

Fix build on !clang:
app.cc:309:39: error: cannot convert 'const GdkRGBA {aka const _GdkRGBA} to 'gdouble {aka double}' in initialization

cf https://blog.quasardb.net/cpp-braced-initialization/

Index: src/app/app.cc
--- src/app/app.cc.orig
+++ src/app/app.cc
@@ -308,7 +308,7 @@ class Options { (public)
                 else
                         alpha = get_alpha();
 
-                GdkRGBA color{bg_color};
+                GdkRGBA color = bg_color;
                 color.alpha = alpha;
                 return color;
         }
